src/main/java/org/zoo/Animals.java

```diff
@@ -1,4 +1,4 @@
-package org.example;
+package org.zoo;
 public abstract class Animals extends EarthCreatures {
 
     public Animals () {
```

Comment:
Is the constructor necessary here with the category? I don't see it used anywhere